### DISCUSSION TITLE
Does this java library provide APIs to work with git lfs

### DISCUSSION THREAD

**Discussion by mantri-mounika** [hub4j/github-api#1379] [Q&A] [answered]
Hi Team,

We are exploring java client which to connect to git server and perform git lfs operations like acquire the lock and release the lock.
We see the REST APIs instead we are looking for java library. Does this java library provide APIs to work with git lfs .Could you please help us.

**Comment by gsmet** [comment, marked as answer]
This library is about the GitHub REST API, not Git.

If you need a library for Git, have a look at JGit: https://www.eclipse.org/jgit/
